src/current/_includes/v22.1/misc/enterprise-features.md

```diff
@@ -1,10 +1,19 @@
+### Cluster optimization
+
+[Follower Reads](follower-reads.html) | Reduce read latency in multi-region deployments by using the closest replica at the expense of reading slightly historical data.
```

Comment:
Just a heads up that this is not rendering correctly, I suppose because there is no table header.